### PR TITLE
Improve test coverage

### LINKED ISSUE #66: Improve Test Coverage
Now that codecov works, we might as well use it :smile: - it looks like most of the uncovered lines should be easy to add tests for anyway

### PR DESCRIPTION
**Pull request by rossviljoen**
Adds tests to cover some currently uncovered lines. Closes #66

### CODE REVIEW

File: test/expected_loglik.jl [hunk 1]
```diff
@@ -37,4 +37,12 @@
         results = map(m -> ApproximateGPs.expected_loglik(m, y, q_f, lik), methods)
         @test all(x -> isapprox(x, results[end]; atol=1e-6, rtol=1e-3), results)
     end
+
+    @test ApproximateGPs.expected_loglik(
+        MonteCarlo(), zeros(10), q_f, GaussianLikelihood()
+    ) isa Real
+    @test ApproximateGPs.expected_loglik(
```
Comment (willtebbutt) [inline test/expected_loglik.jl:44]: Could you make this test check that this result is close to the exact version?

**Review by willtebbutt — COMMENTED** [review]
Just one minor thing, otherwise this looks great.

File: test/expected_loglik.jl (hunk 1 above)
Comment (rossviljoen) [reply test/expected_loglik.jl:44]: That's already checked in the tests above

Comment (willtebbutt) [reply test/expected_loglik.jl:44]: Oh, right, yes. But then why does it also need to be tested here? Which line is it covering that wasn't previously covered?

Comment (rossviljoen) [reply test/expected_loglik.jl:44]: It's for the default constructors `MonteCarlo()` and `GaussHermite()`, which don't _really_ need to be tested separately but I suppose there's no reason not to

Comment (willtebbutt) [reply test/expected_loglik.jl:44]: Ohhhhhhhhhhh I see. Yeah, could to ensure that they're covered. I'm happy with these tests.

**Review by willtebbutt — APPROVED** [review]
I'm happy with this. Please merge when you're happy.

**Comment by rossviljoen** [comment]
I might still add some tests for `laplace.jl`, but codecov isn't currently working for me...

**Pull request merged**